### PR TITLE
trajectory class cleanup 

### PR DESCRIPTION
**Pull request by venkatkapil24**
Simple PR that addresses a bug in forces_component_raw for nbeads>1. Currently, ring polymer interpolation is not being applied.

### CODE REVIEW

File: ipi/engine/properties.py
Comment (venkatkapil24) [inline ipi/engine/properties.py:1]: @ceriottm line 2942 also needs to be changed, right?

File: ipi/engine/properties.py [hunk 1]
```diff
@@ -2934,9 +2934,9 @@ def __init__(self):
                        with the specified bead (with the level of discretization of the component), otherwise the 
                        centoid force is computed. The weight of the potential is not applied. """,
                 "func": lambda index, bead="-1": (
-                    self.system.forces.forces_component(int(index), False, False).sum(
-                        axis=0
-                    )
+                    self.system.forces.forces_component(
+                        int(index), weighted=False, interpolate=True
+                    ).sum(axis=0)
```
Comment (ceriottm) [inline ipi/engine/properties.py:2939]: Set interpolate to false, remove the division by nbeads (that should be for the component, and not the global one) and instead use mean

File: ipi/engine/properties.py
Comment (ceriottm) [reply ipi/engine/properties.py:1]: you mean 2937? yes I'd say, to .mean()

File: ipi/utils/io/__init__.py [hunk 2]
```diff
@@ -67,7 +68,7 @@ def _get_io_function(mode, io):
     available ipi/utils/io/backends/io_*.py backends.
 
     Args:
-        mode: Which format has the file? e.g. "pdb", "xml" or "xyz"
+        mode: Which format has the file? e.g. "pdb", "xml", "bin", "extxyz", or "xyz"
```
Comment (ceriottm) [inline ipi/utils/io/__init__.py:71]: this is "ase" related stuff. I wouldn't do a half job here.

**Comment by venkatkapil24** [comment]
I have managed to run the tests locally without problems. Do pytests fail for ASE requiring examples? The failure is for the example I added /home/runner/work/i-pi/i-pi/examples/features/input_output/extxyz_outputs. @litman90  @ceriottm

**Comment by venkatkapil24** [comment]
=================================== FAILURES ===================================
/home/runner/work/i-pi/i-pi/ipi_tests/examples/test_examples.py:38: RuntimeError
=========================== short test summary info ============================
FAILED ipi_tests/examples/test_examples.py::test_example[/home/runner/work/i-pi/i-pi/examples/features/input_output/extxyz_outputs] - RuntimeError
================== 1 failed, 158 passed in 300.32s (0:05:00) ===================
Traceback (most recent call last):
  File "/home/runner/work/i-pi/i-pi/bin/i-pi-tests", line 66, in <module>
    raise RuntimeError("pytest exit code is {}".format(exit_code))
RuntimeError: pytest exit code is 1

I also can't figure out much from the failure report.

**Comment by venkatkapil24** [comment]
@ceriottm can you review and run the features/input_output/extxyz_outputs/ example and check it works on your end? I have added it to the exclude list.

**Comment by mahrossi** [comment]
Hi sorry, i missed this request to review. I try this afternoon or something. Is ase installed in the pip? If it is, then we can test this, which would be better.

**Review by ceriottm — APPROVED** [review]

**Pull request merged**